RecoBTag/Configuration/python/RecoBTag_EventContent_cff.py

```diff
@@ -87,9 +91,13 @@
         'keep *_pfCombinedInclusiveSecondaryVertexV2BJetTags_*_*',
         'keep *_pfCombinedMVABJetTags_*_*',
         'keep *_pfCombinedMVAV2BJetTags_*_*',
-	'keep *_pfCombinedSecondaryVertexSoftLeptonBJetTags_*_*',
-##	'keep *_pfCombinedSecondaryVertexSoftLeptonCtagLJetTags_*_*',
-        'keep *_inclusiveCandidateSecondaryVertices_*_*'
+        'keep *_pfCombinedSecondaryVertexSoftLeptonBJetTags_*_*',
+        ##	'keep *_pfCombinedSecondaryVertexSoftLeptonCtagLJetTags_*_*',
```

Comment:
I think thought these were commented out just until ctagging code is in place. Should these be uncommented now?